processing/src/main/java/io/druid/query/filter/Filter.java

```diff
@@ -20,10 +20,16 @@
 package io.druid.query.filter;
 
 import io.druid.collections.bitmap.ImmutableBitmap;
+import io.druid.query.BitmapResultFactory;
+import io.druid.query.DefaultBitmapResultFactory;
 import io.druid.segment.ColumnSelector;
 import io.druid.segment.ColumnSelectorFactory;
 
 /**
+ * {@link #getBitmapIndex} and {@link #getBitmapResult} methods both have default implementations, delegating to each
+ * other. Every implementation of {@link Filter} should override {@link #getBitmapResult}, currently it has a default
+ * implementation for compatibility with Filters in extensions. In Druid 0.11 {@link #getBitmapResult} is going to
```

Comment:
hmmm... I'm not sure if we have a good way to "not forget about" this intention, can you file an issue related to this with a 0.11 tag?